packages/material-ui-styles/src/makeStyles/makeStyles.test.js

```diff
@@ -244,6 +244,22 @@ describe('makeStyles', () => {
       assert.deepEqual(sheetsRegistry.registry[0].classes, { root: 'MuiTextField-root' });
     });
 
+    it('should apply string styles', () => {
```

Comment:
- Shouldn't the test uses `jss-plugin-template`?
- I'm not sure that we need a test for it. We are very likely to defer all these logic to a third-party in the future.

Reply:
You're right, I wasn't using `jss-plugin-template`, and the test was completely pointless.

Fixed it now, with a couple of rules in a string to see it uses `jss` properly, I'm not sure if it's useful since this is an external library. If it's not useful, I will remove the test.